test/elixir_console_web/features/console_test.exs

```diff
@@ -23,15 +26,15 @@ defmodule ElixirConsoleWeb.ConsoleTest do
   feature "visitor can get suggestions while typing", %{session: session} do
     session
     |> visit("/")
-    |> fill_in(css("#commandInput"), with: "Enu")
+    |> fill_in(@command_input, with: "Enu")
     |> send_keys([:tab])
     |> find(css(".test-suggestion-list"), fn suggestions_list ->
```

Comment:
This one could also be a module attribute

Reply:
Sounds good - done